platform/ios/app/MBXViewController.m

```diff
@@ -5,6 +5,7 @@
 #import "MBXOfflinePacksTableViewController.h"
 #import "MBXAnnotationView.h"
 #import "MBXUserLocationAnnotationView.h"
+#import "NSValue+MGLStyleEnumAttributeAdditions.h"
```

Comment:
Import this header in both SDKs’ umbrella headers instead of here, since only iosapp and macosapp have this file in their header search paths.